src/gu/sceGuInit.c

```diff
@@ -235,6 +235,25 @@ static unsigned int __attribute__((aligned(16))) ge_init_list[] =
 	ZV(END),
 };
 
+static void callbackFin(int id, void* arg)
```

Comment:
```suggestion
void callbackFin(int id, void* arg)
```
This cannot be static, otherwise it won't match the definition in `guInternal.h`.